api/src/test/java/com/epam/pipeline/test/creator/cluster/NodeCreatorUtils.java

```diff
@@ -32,14 +36,30 @@
 import java.time.format.DateTimeFormatter;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 
 import static com.epam.pipeline.test.creator.CommonCreatorConstants.ID;
 import static com.epam.pipeline.test.creator.CommonCreatorConstants.TEST_STRING;
 import static com.epam.pipeline.test.creator.CommonCreatorConstants.TEST_STRING_LIST;
 
 public final class NodeCreatorUtils {
 
+    public static final TypeReference<Result<List<MasterNode>>> MASTER_NODE_LIST_TYPE =
+            new TypeReference<Result<List<MasterNode>>>() { };
+    public static final TypeReference<Result<List<NodeInstance>>> NODE_INSTANCE_LIST_TYPE =
+            new TypeReference<Result<List<NodeInstance>>>() { };
+    public static final TypeReference<Result<NodeInstance>> NODE_INSTANCE_TYPE =
+            new TypeReference<Result<NodeInstance>>() { };
+    public static final TypeReference<Result<List<InstanceType>>> INSTANCE_TYPE_LIST_TYPE =
+            new TypeReference<Result<List<InstanceType>>>() { };
+    public static final TypeReference<Result<AllowedInstanceAndPriceTypes>> ALLOWED_INSTANCE_TYPE =
+            new TypeReference<Result<AllowedInstanceAndPriceTypes>>() { };
+    public static final TypeReference<Result<List<MonitoringStats>>> MONITORING_STATS_TYPE =
+            new TypeReference<Result<List<MonitoringStats>>>() { };
+    public static final TypeReference<Result<List<NodeDisk>>> NODE_DISK_TYPE =
```

Comment:
Let's use the same naming approach here. It has to be `NODE_DISK_LIST_TYPE` or `NODE_DISKS_TYPE`.